oneview/resource_server_profile_template.go

```diff
@@ -1901,11 +1928,12 @@ func resourceServerProfileTemplateRead(d *schema.ResourceData, meta interface{})
 		mpSettings := make([]map[string]interface{}, 0)
 		if len(spt.ManagementProcessor.MpSettings) != 0 {
 			// initializing schema variables...
-			adminAcc := make([]map[string]interface{}, 0)
-			directory := make([]map[string]interface{}, 0)
-			keyManager := make([]map[string]interface{}, 0)
+			adminAcc := make([]map[string]interface{}, 1)
```

Comment:
any reason why changed to one ?

Reply:
Since we have maxItems as 1, giving length also as 1 and allocating memory for only one item. If its 0, you can append n no of elements to it.